src/Sfa.Tl.ResultsAndCertification.Api.Client/Clients/DfeSignInApiClient.cs

```diff
@@ -39,27 +42,32 @@ public async Task<DfeUserInfo> GetDfeSignInUserInfo(string organisationId, strin
 
             var userInfoResult = userInfo.Result;
             var ukprn = organisationUkprn.Result;
+            userInfoResult.Organisation = organisationUkprn.Result.Name;
 
-            if(ukprn.HasValue)
-                userInfoResult.Ukprn = ukprn;
+            if (ukprn.UKPRN.HasValue)
+                userInfoResult.Ukprn = ukprn.UKPRN;
+            else if (!ukprn.UKPRN.HasValue) userInfoResult.HasAccessToService = HasAccesstoService(Common.Constants.OrganisationConstants.AdminOrganisation, userInfoResult, RolesExtensions.AdminDashboardAccess);
```

Comment:
This is redundant. The else statement would imply the ukprn doesn't have a value

Reply:
updated